src/main/java/Decide.java

```diff
@@ -103,7 +95,7 @@ public void calculateCMV(Point[] pts, Parameters params){
         for (int i = 0; i < 15; i++) {
             if (!PUV[i]) {
```

Comment:
These could change order and use the first as the default case in else. Not necessary though 👍